lib/scala/kg/src/main/scala/stores/kg/neo4j/Neo4jKgQueryStore.scala

```diff
@@ -159,14 +159,10 @@ final class Neo4jKgQueryStore @Inject()(configuration: Neo4jStoreConfiguration)
     }
 
 
-    final override def getMatchingNodes(limit: Int, offset: Int, query: KgNodeQuery): List[KgNode] = {
+    final override def getMatchingNodes(limit: Int, offset: Int, query: KgNodeQuery, sorts: Option[List[KgNodeSort]]): List[KgNode] = {
       val cypher = KgNodeQueryCypher(query)
       transaction.run(
-        s"""${cypher}
-           |RETURN ${nodePropertyNamesString}
-           |SKIP ${offset}
-           |LIMIT ${limit}
-           |""".stripMargin,
+        (List(cypher) ++ List(s"RETURN ${nodePropertyNamesString}") ++ (if (sorts.nonEmpty) List(s"ORDER by ${sorts.get.map(sort => s"node.${sort.field.toString} ${sort.direction.toString}").mkString(", ")}") else List()) ++ List(s"SKIP ${offset}") ++ List(s"LIMIT ${limit}")).mkString("\n"),
```

Comment:
You took a nice multi-line string and made it unreadable..
It's fine if the Cypher has a blank line in it.
Something like:
${sorts.map(sorts => s"ORDER ...").getOrElse("")}

Reply:
Yeah that looks a lot better, changed